app/plugin_hooks_test.go

```diff
@@ -816,6 +816,60 @@ func TestUserHasLoggedIn(t *testing.T) {
 	}
 }
 
+func TestUserWasCreated(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+
+	tearDown, _, _ := SetAppEnvironmentWithPlugins(t,
+		[]string{
+			`
+		package main
+
+		import (
+			"github.com/mattermost/mattermost-server/plugin"
+			"github.com/mattermost/mattermost-server/model"
+		)
+
+		type MyPlugin struct {
+			plugin.MattermostPlugin
+		}
+
+		func (p *MyPlugin) UserWasCreated(c *plugin.Context, user *model.User) {
+			user.Nickname = "plugin-callback-success"
+			p.API.LogDebug(c.SessionId)
+			p.API.LogInfo(c.RequestId)
+			p.API.LogError(c.IpAddress)
+			p.API.LogWarn(c.AcceptLanguage)
```

Comment:
Are these log messages needed for the test? If not, could you remove them please?

Reply:
Done.